app/controllers/ViewsController.scala

```diff
@@ -12,14 +12,13 @@ import util.{Acl, Encryption}
 class ViewsController(val acl: Acl, assetsManager: AssetsManager, isDev: Boolean,
                       crypto: Encryption, val deps: BaseFaciaControllerComponents)(implicit ec: ExecutionContext) extends BaseFaciaController(deps) {
 
-  private def shouldRedirectToV2(request: UserRequest[AnyContent]): Boolean = {
-    val shouldRedirectByQs: Boolean = request.queryString.getOrElse("redirect", Seq("true")).contains("true")
-
-    // TODO set correct date
-    val redirectDay = DateTime.parse("2019-12-25")
-    val shouldRedirectByDate: Boolean = DateTime.now().isAfter(redirectDay)
-
-    shouldRedirectByQs && (isDev || shouldRedirectByDate)
+  private def shouldRedirectToV2(request: UserRequest[AnyContent], priority: Option[String] = None): Boolean = {
+    val isBreakingNews = priority.getOrElse("") == "breaking-news" || request.queryString.getOrElse("layout", Seq("")).exists(_.contains("breaking-news"))
```

Comment:
curious to know how `/breaking-news` works w/out an explicit play route...

Reply:
it does have a route in `routes` -- see `VanityRedirects.scala`